RadixWallet/Features/HomeFeature/Children/NPSSurvey/NPSSurvey+View.swift

```diff
@@ -0,0 +1,127 @@
+// MARK: - NPSSurvey.View
+extension NPSSurvey {
+	public struct View: SwiftUI.View {
+		public let store: StoreOf<NPSSurvey>
+
+		public var body: some SwiftUI.View {
+			WithPerceptionTracking {
+				NavigationStack {
+					ScrollView {
+						VStack(spacing: .zero) {
+							CloseButton { store.send(.view(.closeButtonTapped)) }
+								.flushedLeft
+								.padding(.top, .medium3)
+
+							VStack(spacing: .zero) {
+								headerView()
+								scoreSelectionView()
+
+								Divider()
+									.padding(.bottom, .medium1)
+
+								scoreReasonView()
+							}
+							.padding([.horizontal, .bottom], .large3)
+						}
+					}
+					.separator(.top)
+					.footer {
+						WithControlRequirements(
+							store.feedbackScore,
+							forAction: { store.send(.view(.submitFeedbackTapped(score: $0))) }
+						) { action in
+							Button("Submit Feedback - Thanks!", action: action)
+								.buttonStyle(.primaryRectangular)
+								.controlState(store.isUploadingFeedback ? .loading(.local) : .enabled)
+						}
+					}
+					.presentationDragIndicator(.visible)
+				}
+			}
+		}
+	}
+}
+
+extension NPSSurvey.View {
+	@ViewBuilder
+	private func headerView() -> some SwiftUI.View {
+		Text("How’s it Going?")
+			.textStyle(.sheetTitle)
+			.foregroundStyle(.app.gray1)
+			.padding(.bottom, .medium3)
+
+		Text("How likely are you to recommend Radix and the Radix Wallet to your friends or colleagues?")
+			.multilineTextAlignment(.center)
+			.textStyle(.body1Regular)
+			.foregroundStyle(.app.gray1)
+			.padding(.bottom, .large2)
+	}
+
+	@ViewBuilder
+	private func scoreSelectionView() -> some SwiftUI.View {
+		VStack(alignment: .center, spacing: .small2) {
+			HStack {
+				ForEach(0 ..< 6) { score in
+					scoreButton(score)
+				}
+			}
+
+			HStack {
+				ForEach(6 ..< 11) { score in
```

Comment:
I prefer `6 ... 10`, i.e. to not show out of bounds values (11).

Reply:
but maybe use a constant for the NPS scale 0-10, and use it everywhere, seems best? i.e. here but also strings, i.e. "0 - Not Likeley" etc. (yes I know, it wont change..., but is semantically most correct)